### ISSUE TITLE
Bower install should automatically install both files

### ISSUE THREAD

**Issue by jammed343** [cwill747/angular-libphonenumber#75]
## Expected Behavior

bower install --save angular-libphonenumber
Should include both files with tools like gulp / wiredep
- angular-libphonenumber.js
- libphonenumber.js
## Current Behavior

Currently it just includes
- angular-libphonenumber.js
## Possible Solution

Add libphonenumber.js to main in bower.json
## Steps to Reproduce (for bugs)
1. Use gulp/wiredep
2. bower install --save angular-libphonenumber
3. Error about phoneUtils

**Comment by cwill747** [comment]
The only problem with this approach is what I have [here](https://github.com/cwill747/angular-libphonenumber#installation). I give the option of _choosing_ which `libphonenumber` you want to include, which this would prevent. Should we just set a sane default?

**Comment by jammed343** [comment]
@cwill747 

Ideally we would be able to include `libphonenumber` as a dependency instead, and not have it edited and included in this project. 

I tried using https://github.com/nathanhammond/libphonenumber as an example but It didnt work, had an error of a function could not be found.

**Comment by cwill747** [comment]
Yeah - I use a fork of his library to include the function that's needed for this lib, hence the edited version. Maybe I can just published the edited version as a scoped package or something...

**Comment by jammed343** [comment]
Ive only just been introduced to Google libphonenumber.
I assumed there would be one version that could be relied upon in bower.
I'm not sure why Google packages all languages together in https://github.com/googlei18n/libphonenumber

So I'm not sure if its best to just save the modified https://github.com/nathanhammond/libphonenumber separately or pull your changes into his repo.
In an ideal world libphonenumber could just come directly from google.

**Comment by jlf123** [comment]
I would also like to see this issue resolved.

**Comment by yeluolei** [comment]
wish to see this problem resolved. upgrade to new version of libphonenumber is too hard now.
